### ISSUE TITLE
Add space_after_anon_function option support to Pretty Diff beautifier

### ISSUE THREAD

**Issue by abhishekdev** [Glavin001/atom-beautify#410]
On beautification of JSX files, atom-beautify inserts a space between the `function` keyword and the opening braces. This leads to an error being reported in linter-jscs. The linter is checking against the google jscs preset.

I cant find a way to avoid the beautifier inserting the `space` character. This might be a configuration issue with pretty-diff (since thats the only beautifier for jsx files). Pretty diff API seems to have a [space](http://prettydiff.com/documentation.xhtml#space) configuration that controls the space after the JavaScript function. Its true by default. Is there a way to set it to false?

Before beautification:
```jsx
var HelloWorld = React.createClass({
    render: function() {
        return (
            <h1>{this.props.nls.hello}</h1>
        );
    }
});
```

After beautification:

```jsx
var HelloWorld = React.createClass({
    render: function () {
        return (
            <h1>{this.props.nls.hello}</h1>
        );
    }
});
```

**Comment by Glavin001** [comment]
Looks like the `space` option is not yet added to Atom Beautify. See https://github.com/Glavin001/atom-beautify/blob/master/src/beautifiers/prettydiff.coffee#L9-L28

I would review and accept a Pull Request that added the `space` option.

- [x] Add option to languages. See https://github.com/Glavin001/atom-beautify/blob/master/src/languages/javascript.coffee#L67
- [ ] Add option to `PrettyDiff` beautifier:  https://github.com/Glavin001/atom-beautify/blob/master/src/beautifiers/prettydiff.coffee#L8
  - [ ] Use rename type of option, from `space_after_anon_function` to Pretty Diff's `space` option
    - it would be similar to `inchar: "indent_char"` which is `indent_char` being renamed as `inchar` before being passed to Pretty Diff:
https://github.com/Glavin001/atom-beautify/blob/master/src/beautifiers/prettydiff.coffee#L9
    - so it would be `space: "space_after_anon_function"`

Thanks!

**Comment by Glavin001** [comment]
Published to v0.28.6

**Comment by abhishekdev** [comment]
:+1: Awesome. Thanks!
